src/index.js

```diff
@@ -1,17 +1,22 @@
-import React from 'react';
-import ReactDOM from 'react-dom/client';
-import './index.css';
-import App from './App';
-import reportWebVitals from './reportWebVitals';
-
-const root = ReactDOM.createRoot(document.getElementById('root'));
+import React from "react";
+import ReactDOM from "react-dom/client";
+import "./index.css";
+import App from "./App";
+import { BrowserRouter, Routes, Route } from "react-router-dom";
+import SuitsQuiz from "./pages/SuitsQuiz";
+import MarvelQuiz from "./pages/MarvelQuiz";
+import { QuizContextProvider } from "./quizReducer";
+import Results from "./pages/Results";
+const root = ReactDOM.createRoot(document.getElementById("root"));
 root.render(
-  <React.StrictMode>
-    <App />
-  </React.StrictMode>
+  <BrowserRouter>
+    <QuizContextProvider>
+      <Routes>
+        <Route path="/" element={<App />} />
+        <Route path="/suits-quiz" element={<SuitsQuiz />} />
+        <Route path="/marvel-quiz" element={<MarvelQuiz />} />
+        <Route path="/results" element={<Results />} />
```

Comment:
not good way keep only app here
and the structure App
with layout and other stuff